experiment_buddy/experiment_buddy.py

```diff
@@ -219,11 +223,15 @@ def _ask_experiment_id(cluster, sweep):
         experiment_id = tkinter.simpledialog.askstring(title, "experiment_id")
         root.destroy()
     except:
-        experiment_id = input(f"Running on {title} \ndescribe your experiment (experiment_id):\n")
+        if os.environ['BRANCH_TESTING']:
+            import uuid
+            experiment_id = f'BRANCH_TESTING-{os.environ["BRANCH_TESTING"]}-{uuid.uuid4()}'
+        else:
+            experiment_id = input(f"Running on {title}\ndescribe your experiment (experiment_id):\n")
 
     experiment_id = (experiment_id or "no_id").replace(" ", "_")
     if cluster:
-        experiment_id = f"[CLUSTER] {experiment_id}"
+        return f"[CLUSTER] {experiment_id}"
```

Comment:
Don't change previous code unless it's important

While i usually agree that returning early is a good pattern, here the gain is null